axum-extra/src/routing/mod.rs

```diff
@@ -178,7 +178,7 @@ pub trait RouterExt<S, B>: sealed::Sealed {
 impl<S, B> RouterExt<S, B> for Router<S, B>
 where
     B: axum::body::HttpBody + Send + 'static,
-    S: Send + Sync + 'static,
+    S: Clone + Send + Sync + 'static,
```

Comment:
I'm pretty sure `Sync` is now no longer needed, and one of `Send` or `'static` might not be either. We can relax these in a separate PR though, if you prefer.